sequence_processing_pipeline/tests/data/configuration_profiles/miseq_metagenomic.json

```diff
@@ -14,6 +14,38 @@
         "executable_path": "bcl2fastq",
         "per_process_memory_limit": "100gb"
       },
+      "bcl-convert": {
+        "nodes": 1,
+        "nprocs": 16,
+        "queue": "qiita",
+        "wallclock_time_in_minutes": 216,
+        "modules_to_load": [
+          "bclconvert_3.7.5"
+        ],
+        "executable_path": "bcl-convert",
+        "per_process_memory_limit": "10gb"
+      },
+      "qc": {
```

Comment:
should these section be deleted?

Reply:
debated that myself. IMOO, it should be retained as long as the legacy QC implementation is still in the codebase. The question is really whether NuQC is stable enough that we can remove legacy QC from the current codebase. My vote is yes.

Reply:
Agree and vote for a different PR for that.